pkg/model/dev.go

```diff
@@ -700,21 +715,28 @@ func (dev *Dev) ToTranslationRule(main *Dev) *TranslationRule {
 		rule.Args = []string{}
 	}
 
-	for _, v := range dev.Volumes {
-		var devSubPath string
-		if v.SubPath == "" {
-			devSubPath = fullGlobalSubPath(v.MountPath)
-		} else {
-			devSubPath = fullDevSubPath(v.SubPath)
+	if main.PersistentVolumeEnabled() {
+		for _, v := range dev.Volumes {
+			if v.LocalPath == "" {
```

Comment:
can we create the volume and just change the subpath once to avoid declaring the whole thing twice?